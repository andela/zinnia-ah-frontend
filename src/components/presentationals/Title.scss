@import url("https://fonts.googleapis.com/css?family=Lato:400,700|Lora:400,700|Sacramento");
@import "../../assets/variables";

.title {
  color: $primary-black;
  font-family: "Lora", serif;
  font-size: 20px;
  font-weight: bold;
}

.title-lg {
  font-size: 46px;
}

.text-center {
  text-align: center;
}
