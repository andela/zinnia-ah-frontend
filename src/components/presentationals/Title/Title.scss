@import '../../../assets/variables';

.title {
  color: $primary-black;
  font-family: $lora;
  font-size: 1.25rem;
  font-weight: bold;
}

.title-sm {
  font-size: .875rem;
}

.title-lg {
  font-size: 2.875rem;
}

.title-xl {
  font-size: 4rem;
}

.lh-4 {
  line-height: 4rem;
  margin-bottom: 0;
}

.title-md {
  font-size: 1.875rem;
}


.text-center {
  text-align: center;
}
