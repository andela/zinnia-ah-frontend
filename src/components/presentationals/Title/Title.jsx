import React from 'react';
import PropTypes from 'prop-types';

import './Title.scss';

const Title = props => {
  return (
    <h1 className={`title ${props.title.class}`}>{props.title.content}</h1>
  );
};

Title.propTypes = {
  title: PropTypes.shape({
    class: PropTypes.string,
    content: PropTypes.string,
  }),
};

export default Title;
