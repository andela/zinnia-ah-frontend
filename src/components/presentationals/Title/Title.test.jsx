import React from 'react';
import { mount } from 'enzyme';

import Title from './Title';

describe('Title', () => {
  it('should render correctly', () => {
    const component = mount(<Title className="title" content="title" />);
    expect(component).toMatchSnapshot();
  });
});
