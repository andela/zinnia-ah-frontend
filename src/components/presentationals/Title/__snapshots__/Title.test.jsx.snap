// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly 1`] = `
<Title
  className="title"
  content="title"
>
  <h1
    className="title title"
  >
    title
  </h1>
</Title>
`;
